common/src/main/java/com/wynntils/screens/maps/SeaskipperDepartureBoardScreen.java

```diff
@@ -0,0 +1,414 @@
+        // slightly more to the left and all buttons below row 1 are slightly higher
+        int buttonWidth = (int) (Texture.DESTINATION_BUTTON.width() * currentTextureScale);
+        int buttonHeight = (int) (Texture.DESTINATION_BUTTON.height() / 2f * currentTextureScale);
+        int leftX = (int) (5 + (4 * currentTextureScale));
+        int rightX = (int) (7 * currentTextureScale) + 7;
+        int rowOffset = 5;
+        int yOffset = (int) (destinationListY + rowOffset * currentTextureScale);
+        int buttonIndex = 0;
+
+        for (SeaskipperDestinationPoi poi : destinationPois) {
+            if (poi == currentLocationPoi) continue;
+
+            boolean selected = selectedDestination == poi;
+
+            SeaskipperDestinationButton newButton;
+
+            if (buttonIndex == 0) {
+                newButton = new SeaskipperDestinationButton(
+                        leftX,
+                        (yOffset + (destinationButtons.size() * buttonHeight)),
+                        buttonWidth,
+                        buttonHeight,
+                        selected,
+                        poi);
+            } else if (buttonIndex == 1) {
+                newButton = new SeaskipperDestinationButton(
+                        (rightX + buttonWidth),
+                        (yOffset + ((destinationButtons.size() - 1) * buttonHeight)),
+                        buttonWidth,
+                        buttonHeight,
+                        selected,
+                        poi);
+
+                rowOffset += 2;
+                yOffset = (int) (destinationListY + rowOffset * currentTextureScale);
+            } else if (buttonIndex % 2 == 0) {
+                newButton = new SeaskipperDestinationButton(
+                        leftX,
+                        (yOffset + ((destinationButtons.size() - (buttonIndex / 2)) * buttonHeight)),
+                        buttonWidth,
+                        buttonHeight,
+                        selected,
+                        poi);
+            } else {
+                newButton = new SeaskipperDestinationButton(
+                        (rightX + buttonWidth),
+                        (yOffset + ((destinationButtons.size() - (buttonIndex / 2) - 1) * buttonHeight)),
+                        buttonWidth,
+                        buttonHeight,
+                        selected,
+                        poi);
+
+                rowOffset += 2;
+                yOffset = (int) (destinationListY + rowOffset * currentTextureScale);
+            }
+
+            buttonIndex++;
+
+            destinationButtons.add(newButton);
+        }
+    }
+
+    private void zoomToDestination(Poi destination) {
+        updateMapCenter(
+                destination.getLocation().getX(), destination.getLocation().getZ());
+
+        float zoomLevel = McUtils.mc().getWindow().getGuiScaledHeight() / 650f;
```

Comment:
What is the magic constant of 650 mean? How did you get it? Please extract it into `private static final` to the top of the class.